lte/gateway/c/oai/tasks/s1ap/s1ap_mme_handlers.c

```diff
@@ -2342,6 +2460,100 @@ int s1ap_mme_handle_handover_required(
   OAILOG_FUNC_RETURN(LOG_S1AP, RETURNok);
 }
 
+int s1ap_mme_handle_handover_command(
+    s1ap_state_t* state, const itti_mme_app_handover_command_t* ho_command_p) {
+  uint8_t* buffer_p   = NULL;
+  uint8_t err         = 0;
+  uint32_t length     = 0;
+  S1ap_S1AP_PDU_t pdu = {0};
+  S1ap_HandoverCommand_t* out;
+  S1ap_HandoverCommandIEs_t* ie = NULL;
+  ue_description_t* ue_ref_p    = NULL;
+  sctp_stream_id_t stream       = 0x0;
+
+  OAILOG_FUNC_IN(LOG_S1AP);
+  if (ho_command_p == NULL) {
+    OAILOG_ERROR(LOG_S1AP, "Handover Command is null\n");
+    OAILOG_FUNC_RETURN(LOG_S1AP, RETURNerror);
+  }
+
+  if ((ue_ref_p = s1ap_state_get_ue_mmeid(ho_command_p->mme_ue_s1ap_id)) ==
+      NULL) {
+    OAILOG_ERROR(
+        LOG_S1AP,
+        "could not get ue context for mme_ue_s1ap_id " MME_UE_S1AP_ID_FMT
+        ", using 0x0!\n",
+        (uint32_t) ho_command_p->mme_ue_s1ap_id);
+    stream = (sctp_stream_id_t) 0x0;  // Seems to work as a fallback...
```

Comment:
Looks like a flaky way of handling this error. When, why we end up a removed UE context and why shall we continue with the HO command?